### PR TITLE
Fix #203: installer for nersc development

### LINKED ISSUE #203: make installer for nersc pyenv
- bashrc.ext
- relevant bits from https://github.com/radiasoft/download/blob/master/installers/nersc-sirepo-update/radiasoft-download.sh
- PYTHON_CONFIGURE_OPTS="--enable-shared" pyenv install 3.7.2

### PR DESCRIPTION
**Pull request by e-carlin**

### CODE REVIEW

File: installers/nersc-dev/radiasoft-download.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#
+# To run: curl radia.run | bash -s nersc-dev
+#
+nersc_dev_main() {
+    local r=~/.pyenv
+    if [[ ! -d $r ]]; then
+        git clone https://github.com/pyenv/pyenv.git "$r"
```
Comment (robnagler) [inline installers/nersc-dev/radiasoft-download.sh:8]: why not use `curl https://pyenv.run | bash`?

File: installers/nersc-dev/radiasoft-download.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    if [[ ! -e $r/versions/$v ]]; then
+        PYTHON_CONFIGURE_OPTS="--enable-shared" pyenv install "$v"
+    fi
+    local a=py3
+    if [[ ! -e $r/versions/$a ]]; then
+        pyenv virtualenv "$v" "$a"
+    fi
+    pyenv global "$a"
+    if ! grep 'pyenv init' ~/.bashrc.ext; then
+        perl -pi -e '/exiting .bashrc.ext/ && ($_ = q{
```
Comment (robnagler) [inline installers/nersc-dev/radiasoft-download.sh:26]: JAPH!

**Review by robnagler — CHANGES_REQUESTED** [review]
I think this should be `nersc-pyenv`, not `nersc-dev`.

File: installers/nersc-dev/radiasoft-download.sh (hunk 1 above)
Comment (e-carlin) [reply installers/nersc-dev/radiasoft-download.sh]: 8ffe45a

**Review by robnagler — APPROVED** [review]
I'll let you merge.

**Pull request merged**